src/viewport.js

```diff
@@ -79,6 +84,12 @@ export class Viewport {
     /** @private @const {!Observable<!ViewportChangedEvent>} */
     this.changeObservable_ = new Observable();
 
+    /** @private {?HTMLMetaElement|undefined} */
+    this.viewportMeta_ = undefined;
```

Comment:
any reason why we need it to be both nullable and undefined? (i know probably on some assignments it might be either one or the other?)